cse-lib/data/src/main/java/com/farao_community/farao/cse/data/ttc_res/TtcResult.java

```diff
@@ -204,7 +207,7 @@ private static void fillBordersExchanges(ProcessData processData, Timestamp ttcR
             processData.borderExchanges.forEach((boundary, exchangedFlow) -> {
                 BorderExchange borderExchange = new BorderExchange();
                 Border border = new Border();
-                border.setV(boundary);
+                border.setV(boundary.replace(" ", ""));
```

Comment:
I think it's better to change the keys in the borderExchanges map and put the right keys instead of modifying them afterwards.

Reply:
done